Formula/cjdns.rb

```diff
@@ -18,7 +18,17 @@ class Cjdns < Formula
   depends_on "node" => :build
   depends_on "rust" => :build
 
+  on_macos do
+    # Fails to build with python@3.10.
+    # AttributeError: module 'collections' has no attribute 'MutableSet'
+    # Related PR: https://github.com/cjdelisle/cjdns/pull/1246
+    depends_on "python@3.9" => :build
+  end
+
   def install
+    # Libuv build fails on macOS with: env: python: No such file or directory
+    ENV.prepend_path "PATH", Formula["python@3.9"].opt_libexec/"bin" if OS.mac?
```

Comment:
Checking macOS-only first as not sure if `python` executable was ever on PATH in Linux CI.